bin/meta-git-remote

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const program = require('commander');
+const debug = require('debug')('meta-git-remote');
+const loop = require('loop-things');
+const path = require('path');
+const util = require('util');
+
+const usages = `\n  usages:\n
+    meta git remote
+    meta git remote -v
+    meta git remote show origin
+    meta git remote rename oldname newname
+    meta git remote rm name`;
+
+if (process.argv[2] === '--help')
+  return console.log(usages);
+
+let first = process.argv[2] === 'blank' ?
+                      process.argv[3] || '':
+                      process.argv[2] || '';
+
+const subCommandParsers = {
+  '-v': function (argv) { return '' },
+  'show': function (argv) {
+    let remote = process.argv[3] === 'blank' ?
+                      process.argv[4] || '':
+                      process.argv[3] || '';
+    return `show ${remote}`;
+  },
+  'rename': function (argv) {
+    
+  },
+  'rm': function (argv) {}
+}
+
+const subCommandParser = subCommandParsers[first];
+
+if ( ! subCommandParser) return console.log(usages);
```

Comment:
Probably make sense to force them to type `meta git remote --help` for usage, in this case. I really just wanted to get a skeleton out there to show you how easy it is to make these commands and see if you'd be interested in helping to develop them. I put this together in just a couple minutes. I'd be you'd be able to get an api you'd like equally quickly. Incremental push coming momentarily. 

Happy to make you a contributor, if you're interested, and we can just go back and forth with commits on the PR until we think it looks good.